src/site/content/en/blog/contact-picker/index.md

```diff
@@ -147,10 +123,11 @@ user gesture.
 
 ### Handling the results
 
-The Contact Picker API returns an array of contacts, and each contact
-includes an array of the requested properties. If a contact doesn't have
-data for the requested property, or the user chooses to opt-out of sharing
-a particular property, it returns an empty array.
+The Contact Picker API returns an array of contacts, and each contact includes
+an array of the requested properties. If a contact doesn't have data for the
+requested property, or the user chooses to opt-out of sharing a particular
+property, it returns an empty array. (I describe how the user chooses properties
```

Comment:
```suggestion
property, the API returns an empty array. (I describe how the user chooses properties
```
To avoid a confusing pronoun. (Technically, "it" was referring to "property.")